### ISSUE TITLE
AttributeError: 'UTSample' object has no attribute 'is_legal'

### ISSUE THREAD

**Issue by Nomiluks** [textflint/textflint#12]
Hi, I am unable to run `UT (Universal Transformation)`  pipeline. Details are given below.

**Code:**
```
from textflint import Engine

# load the data samples
sample1 = {'x': 'Titanic is my favorite movie.', 'y': 'a'}
sample2 = {'x': 'I don\'t like the actor Tim Hill', 'y': 'a'}
data_samples = [sample1, sample2]

# define the output directory
out_dir_path = 'test_result/'

engine = Engine('UT')
engine.run(data_samples, out_dir_path)
```

**Error log:**
```
TextFlint: ******Start load!******
100%|██████████| 2/2 [00:00<00:00, 3958.76it/s]
TextFlint: 2 in total, 2 were loaded successful.
TextFlint: ******Finish load!******
TextFlint: ******Start SwapSynWordEmbedding!******
  0%|          | 0/2 [00:00<?, ?it/s]
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-77-c6dd1e85c4e5> in <module>()
----> 1 engine.run(data_samples, out_dir_path)

4 frames
/usr/local/lib/python3.7/dist-packages/textflint/generation_layer/transformation/transformation.py in <listcomp>(.0)
     88         if transform_results:
     89             return [sample for sample in transform_results
---> 90                     if (not sample.is_origin) and sample.is_legal()]
     91         else:
     92             return []

AttributeError: 'UTSample' object has no attribute 'is_legal'
```

**Comment by BeyonderXX** [comment]
Thanks for your feadback !

It's a bug, we will fix it as soon as possible.

If you want to transform data sample in specific task, you can pass task name rather than "UT" to `Engine`. 
TextFlint will apply all adapted **universal transformations** automatically.

**Comment by BeyonderXX** [comment]
We have fixed the bug in 0.0.3 version, thank u !!!
